detox/src/android/espressoapi/UIDevice.js

```diff
@@ -151,31 +151,41 @@ class UiDevice {
   }
 
   static pressKeyCode(element, keyCode) {
-    if (typeof keyCode !== "number") throw new Error("keyCode should be a number, but got " + (keyCode + (" (" + (typeof keyCode + ")"))));
-    return {
-      target: element,
-      method: "pressKeyCode",
-      args: [{
-        type: "Integer",
-        value: keyCode
-      }]
-    };
-  }
-
-  static pressKeyCode(element, keyCode, metaState) {
-    if (typeof keyCode !== "number") throw new Error("keyCode should be a number, but got " + (keyCode + (" (" + (typeof keyCode + ")"))));
-    if (typeof metaState !== "number") throw new Error("metaState should be a number, but got " + (metaState + (" (" + (typeof metaState + ")"))));
-    return {
-      target: element,
-      method: "pressKeyCode",
-      args: [{
-        type: "Integer",
-        value: keyCode
-      }, {
-        type: "Integer",
-        value: metaState
-      }]
-    };
+    function pressKeyCode1(keyCode) {
```

Comment:
What happened here? 
I guess there's problem with Java overloads, did you handle it somewhere else in a different way?
Seems like the original code just has `metaState=0` as default.
https://android.googlesource.com/platform/frameworks/testing/+/79693ed/uiautomator/library/src/com/android/uiautomator/core/UiDevice.java#285

Reply:
The easiest way to handle overloading is to just check for the number of arguments. There are for sure better, more reliable ways, but this was the fastest. (I have checks in place that make sure we get some console output if we mess up here)